graylog2-server/src/main/java/org/graylog2/shared/rest/CORSFilter.java

```diff
@@ -29,7 +29,11 @@
     public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
         // we have already added the necessary headers for OPTIONS requests below
         if (requestContext.getRequest().getMethod().equalsIgnoreCase("options")) {
-            return;
+            if(Response.Status.Family.familyOf(responseContext.getStatus()) == Response.Status.Family.INFORMATIONAL) {
+                return;
+            }
+            responseContext.setStatus(Response.Status.NO_CONTENT.getStatusCode());
+            responseContext.setEntity("");
         }
```

Comment:
This is not working as expected, as there are requests that will get the CORS HTTP headers set twice. Chrome was complaining about that, and the result was that the web interface was not loading even when the server was running. I think we should change it into this:

```
if (Response.Status.Family.familyOf(responseContext.getStatus()) != Response.Status.Family.INFORMATIONAL) {
    responseContext.setStatus(Response.Status.NO_CONTENT.getStatusCode());
    responseContext.setEntity("");
}
return;
```